duckduckgo/Android
# AppTP: Improvements for DeviceShieldActivityFeedFragment

Task/Issue URL: https://app.asana.com/0/1200905986587319/1203352268410406/f

### Description
Updated device shield screen (activity, fragment, adapter, layouts). Fixed sticky headers.

### Steps to test this PR

Check the Asana task (section "Testing - check the attached videos as well" at the bottom of the description) https://app.asana.com/0/1200905986587319/1203352268410406/f and comment https://app.asana.com/0/0/1203352268410406/1203374322675674/f

### NO UI changes
